[master] [cc] Mention about K8 deployment option for production environments (#6087)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim.md

    - Minimum CPU : 4vCPU
    - Minimum Memory : 4GB

!!! note

>     Choreo Connect Docker based deployment option explained here **only for the tryout purpose.** This method is **not recommended to use in production deployments**. For production deployments you can use Kubernetes based Choreo Connect deployment with [Helm artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts/) or [YAML artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane/).

Reviewer: ```suggestion
    The Choreo Connect Docker Compose based deployment option explained here is **only for tryout purposes.** This method is **not recommended for production deployments**. For production deployments, you can use the Kubernetes based Choreo Connect deployment with [Helm artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts/) or [YAML artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane/).
```
